lib/recommend.yaml

```diff
@@ -0,0 +1,50 @@
+linter:
```

Comment:
To echo @devoncarew's comment above: This file should probably contain a comment explaining that it should be used in addition to core.yaml - and not as a substitute.

(That was my first question I had first time looking at these files)

Reply:
Regarding https://github.com/dart-lang/lints/pull/9#discussion_r621696293, should this then import core.yaml and not be used in combination with the other file?

Reply:
Yep.  Done. 👍